css/main.css

```diff
@@ -261,9 +268,89 @@ a:hover {
   background-color: #963dd3;
   color: #fff;
   padding: 10px 20px;
+  border: none;
+  border-radius: 5px;
 }
 
 .last-updated {
   font-size: 11px;
   text-align: center;
-}
\ No newline at end of file
+}
+
+/* Landing page */
+.swing-landing {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  height: 100%;
+  position: relative;
+}
+
+.swing-landing a {
+  text-decoration: underline;
+}
+
+.swing-landing__section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  color: #fff;
+
+  background-position: center;
+  background-size: cover;
+}
+
+.swing-landing__section h2 {
+  z-index: 1;
+}
+
+.swing-landing__section::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #1c1f2c;
+  opacity: 0.6;
+  transition: opacity 0.3s ease-in-out;
+}
+
+.swing-landing__section:hover::before {
+  opacity: 0.3;
+}
+
+.swing-landing__section:hover {
+  color: #fff;
+  cursor: pointer;
+}
+
+.swing-landing__section--brighton {
+  background-image: url("../img/brighton.jpg");
```

Comment:
Update to `https://unsplash.com/photos/blue-and-white-wooden-chairs-near-seaside-WgPC2oG_pYA`